src/ResetMap.jl

```diff
@@ -0,0 +1,169 @@
+import Base: isempty
+
+export ResetMap,
+       apply,
+       apply!
+
+"""
+    ResetMap{N<:Real} <: LazySet{N}
+
+Type that represents a lazy reset map.
```

Comment:
What is the role of the wrapped type `X`? In the example it is not used. Compare to `MathematicalSystems.ResetMap`.

Reply:
OK, perhaps write explicitly: ``ResetMap(X) = \\{x : Ax + b, x \in X\}}`` and add an example of support vector / function evaluation below.